lib/mayaUsd/sceneIndex/proxyShapeSceneIndexPlugin.cpp

```diff
@@ -153,21 +170,30 @@ HdSceneIndexBaseRefPtr MayaUsdProxyShapeMayaNodeSceneIndexPlugin::_AppendSceneIn
             // Flatten transforms, visibility, purpose, model, and material
             // bindings over hierarchies.
             // Do it the same way as USDView does with a SceneIndices chain
-            HdSceneIndexBaseRefPtr _sceneIndex
+            HdSceneIndexBaseRefPtr sceneIndex
                 = UsdImagingPiPrototypePropagatingSceneIndex::New(usdImagingStageSceneIndex);
-            _sceneIndex = UsdImagingNiPrototypePropagatingSceneIndex::New(_sceneIndex);
+#if PXR_VERSION < 2311
+            sceneIndex = UsdImagingNiPrototypePropagatingSceneIndex::New(sceneIndex);
+#else
+            sceneIndex = UsdImagingNiPrototypePropagatingSceneIndex::New(
```

Comment:
If I am correct, this block will never be called as line 137 we already returned from the function for USD 23.11+.